<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="LoginMenu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="Input_Account_EditText">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>手机号或身份证号码</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="Remember_Me_CheckBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>250</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>记住我</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Login_Button">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #0076b4;
    border: 1px solid #dcdfe6;
    padding: 10px;
    border-radius: 5px;
	color: rgb(255, 255, 255);
	font: 10pt &quot;黑体&quot;;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}
</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Register_Button">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>290</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    padding: 10px;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}
</string>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLabel" name="Forgot_Password_Label">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>250</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel:hover { color:rgb(51, 144, 255);}</string>
     </property>
     <property name="text">
      <string>忘记密码？</string>
     </property>
    </widget>
    <widget class="QLabel" name="Welcome_Label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700; color:#0974ae;&quot;&gt;欢迎使用游客预约系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Wrong_Account_Label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt; × 请输入正确的身份证号或手机号！ &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Login_Menu_Image_Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Close_Button">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>0</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-style:outset;
background-color: rgba(255, 255, 255, 0);
border-radius:0px
}

QPushButton:hover{ 
	background-color: rgb(255, 0, 0);
}
 
QPushButton:pressed{
	background-color: rgb(220,92,102);
}</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Hide_Button">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>0</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-style:outset;
background-color: rgba(255, 255, 255, 0);
border-radius:0px
}

QPushButton:hover{ 
	background-color: rgb(239, 231, 231);
}
 
QPushButton:pressed{
	background-color: rgb(243,234,234);
}</string>
     </property>
     <property name="text">
      <string>—</string>
     </property>
    </widget>
    <widget class="QLabel" name="Not_Input_Account_Label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;× 请输入用户名！ &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Not_Input_Password_Label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;  × 请输入密码！ &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Admin_Login_Label">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel:hover { color:rgb(51, 144, 255);}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;管理员登录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Input_Password_EditText">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>200</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>密码</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Wrong_Password_Label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;× 密码错误！ &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Not_Exist_Account_Label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;× 用户名不存在，请先注册账号后登录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <zorder>Input_Password_EditText</zorder>
    <zorder>Input_Account_EditText</zorder>
    <zorder>Remember_Me_CheckBox</zorder>
    <zorder>Login_Button</zorder>
    <zorder>Register_Button</zorder>
    <zorder>Forgot_Password_Label</zorder>
    <zorder>Welcome_Label</zorder>
    <zorder>Wrong_Account_Label</zorder>
    <zorder>Login_Menu_Image_Label</zorder>
    <zorder>Close_Button</zorder>
    <zorder>Hide_Button</zorder>
    <zorder>Not_Input_Account_Label</zorder>
    <zorder>Admin_Login_Label</zorder>
    <zorder>Not_Input_Password_Label</zorder>
    <zorder>Wrong_Password_Label</zorder>
    <zorder>Not_Exist_Account_Label</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>633</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>Input_Account_EditText</tabstop>
  <tabstop>Input_Password_EditText</tabstop>
  <tabstop>Remember_Me_CheckBox</tabstop>
  <tabstop>Login_Button</tabstop>
  <tabstop>Register_Button</tabstop>
  <tabstop>Hide_Button</tabstop>
  <tabstop>Close_Button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>on_Remember_Me_CheckBox_StateChanged()</slot>
 </slots>
</ui>
